lib/services/local/users.go

```diff
@@ -1470,17 +1460,27 @@ func (s *IdentityService) GetKeyAttestationData(ctx context.Context, publicKey c
 		return nil, trace.BadParameter("missing parameter publicKey")
 	}
 
-	sshPub, err := ssh.NewPublicKey(publicKey)
+	pubDER, err := x509.MarshalPKIXPublicKey(publicKey)
 	if err != nil {
 		return nil, trace.Wrap(err)
 	}
 
-	key := ssh.FingerprintSHA256(sshPub)
+	key := fingerprintSha256(pubDER)
 	item, err := s.Get(ctx, backend.Key(attestationsPrefix, key))
-	if err != nil {
+	if trace.IsNotFound(err) {
+		// Fallback to old fingerprint (std base64 encoded ssh public key) for backwards compatibility.
+		// DELETE IN 13.0.0
```

Comment:
```suggestion
		// DELETE IN 13.0, old fingerprints not in use by then (Joerger).
```

Reply:
This key data is added each time you login (with PIV). This backwards compatibility check just makes it so your existing login, which was made using the old fingerprint, could still be found by the auth server.